pkg/controllers/resources/nodes/fake_syncer.go

```diff
@@ -240,3 +240,18 @@ func filterOutDaemonSets(pl *corev1.PodList) []corev1.Pod {
 
 	return podsNoDaemonSets
 }
+
+// Filter away physical DaemonSet Pods using annotations to enable scale down
+func filterOutPhysicalDaemonSets(pl *corev1.PodList) []corev1.Pod {
+	var podsNoDaemoSets []corev1.Pod
```

Comment:
Typo, podsNoDaemoSets -> podsNoDaemonSets.

Reply:
Dammit. It is now fixed :1st_place_medal:

Reply:
Thank you.